packages/theme-chalk/src/select-dropdown-v2.scss

```diff
@@ -11,6 +12,12 @@
   .#{$namespace}-scrollbar.is-empty .#{$namespace}-select-dropdown__list {
     padding: 0;
   }
+
+  @include e(option-item) {
+    &:hover:not(.hover) {
+      background-color: transparent!important;
```

Comment:
Add a space before `!important`, P.S. can we avoid using `!import` statment here?

Reply:
yeah, I avoided using the important statement